packages/SqueakMaps-Core.package/SMAWindow.class/instance/searchMultipleLocations.st

```diff
@@ -1,8 +1,10 @@
 search
 searchMultipleLocations
 
-	(self startSearchInput isEmpty or: [destinationSearchInput isEmpty]) ifTrue: [
+	(self startSearchInput isEmpty or: [self destinationSearchInput isEmpty]) ifTrue: [
 		UIManager default inform: 'Please enter a start and destination location.'.
 		^ self].
-	self tiledMapMorph locate: (self startSearchInput) and: (self destinationSearchInput).
+	self setOpenrouteserviceAsApi.
```

Comment:
Den Aufruf finde ich sehr problematisch. Damit ist es nicht oder nur über Umwege möglich, die routingAPI von außen zu setzen (bspw. für Tests). Wäre es nicht möglich, die API einmal zu setzen, wenn man das SMA Window initialisiert? Also `SMAWindow new. SMAWindow routingAPI: self buildRoutingAPI` oder so ähnlich? Dann kann man in Tests die routing API einfach austauschen und trotzdem Methoden wie `searchMultipleLocations` testen.